PiAUISuite/VoiceCommand/commands.conf

```diff
@@ -0,0 +1,35 @@
+#This is the default config file
+#These are the special options you can set (remove the #)
+#!verify==1
+!keyword==mascot
+#!thresh==0.7
+#!continuous==1
+!response==Hello Sir, having a good day?
+#!quiet==0
+#!ignore==0
+#!filler==0
+#!duration==2
+#!com_dur==3
+#!hardware==plughw:1,0
+#Here are the commands
+show me==/home/pi/AUI/Imaging/test 2
+track me==/home/pi/AUI/Imaging/test 1
+download==download ...
+play $1 season $2 episode $3==playvideo -s $2 -e $3 $1
+download $1 season $2 episode $3==download $1 s$2e$3
+play==playvideo -r -f ...
+multiple==playvideo -r -m -c 5 ...
+download==download ...
+YouTube==youtube-search ...
+Google==google ...
+event== https://thetatausbu.design.blog/fall-rush-2020/
+active== tts "23 active brothers" 2> /dev/null
+time== local time google.com
+restaurant== restaurant near me google.com
+chair== https://thetatausbu.design.blog/executive-board/
```

Comment:
Lines 25-29 looks good on the left hand side of `==`. The right hand side we can leave as place holders. When the LCD tasks get implemented, the right side will be replaced with commands to invoke the scripts from LCD tasks.